packages/main/src/plugin/default-configuration.spec.ts

```diff
@@ -16,15 +16,15 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-import { promises as fsPromises } from 'node:fs';
+import { readFile } from 'node:fs/promises';
 
 import { beforeEach, describe, expect, test, vi } from 'vitest';
 
 import { DefaultConfiguration } from './default-configuration.js';
 import type { Directories } from './directories.js';
 
 // mock the fs module
```

Comment:
```suggestion
// mock the fs promises module
```

Just a suggestion, maybe we can change the comment to this now?